scipy/sparse/linalg/tests/test_propack.py

```diff
@@ -103,21 +98,42 @@ def load_real(folder, precision, file='illc1850.coord'):
     return A
 
 
-def load_complex(folder, precision, file='mhd1280b.cua'):
+def load_complex(folder, precision):
+    file='mhd1280b.cua'
```

Comment:
I went ahead and used some metadata from from the file rather than hard coding, but didn't generalize for all possible files, so it's best not to accept `file` as an argument.

Turns out the `complex8` and `complex16` versions of this file are identical, btw.